tests/test_enterprise/api/test_views.py

```diff
@@ -1998,3 +1998,14 @@ def test_enterprise_customer_course_enrollments_detail_multiple(
         response = self.load_json(response.content)
 
         self.assertListEqual(response, expected_response)
+
+    def test_enterprise_customer_catalogs_response_formats(self):
+        """
+        ``enterprise_catalogs``'s xml and json responses verification.
+        """
+        response_xml_1 = self.client.get('{}?format=xml'.format(ENTERPRISE_CATALOGS_LIST_ENDPOINT))
```

Comment:
Now that we are introducing multiple formats it might make sense to add a line here that requests the default representation and confirms that it is JSON.  Or maybe it's safe enough to assume that the system always returns JSON unless a special case is added, and so we need to only test the special case and not verify the default format has not changed?

Reply:
By default, the content renderer is JSON. Existing tests were validating a json response in test repo. However we can add our test too here to show up the default JSON behavior.